### PR TITLE
Update RTE content when value changes

### PR DESCRIPTION
**Pull request by enkelmedia**
This addresses the issues described here: https://github.com/umbraco/Umbraco.CMS.Backoffice/issues/1136

## Description
I've added implementations to get/set methods for value that will only trigger an update to `editor` after the initial load and when the content is different from the current content of the editor (to avoid calling setContent() with the same content).

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Chore (minor updates related to the tooling or maintenance of the repository, does not impact compiled assets)

## How to test?
There is a simple example-element that can be used for testing here: https://github.com/umbraco/Umbraco.CMS.Backoffice/issues/1136 when I worked on the PR I made these changes to the `umb-property-editor-ui-tiny-mce` locally for testing.

## Checklist

- [x] If my change requires a change to the documentation, I have updated the documentation in this pull request.
- [x] I have read the **[CONTRIBUTING](<(https://github.com/umbraco/Umbraco.CMS.Backoffice/blob/main/.github/CONTRIBUTING.md)>)** document.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: src/packages/core/components/input-tiny-mce/input-tiny-mce.element.ts [hunk 1]
```diff
@@ -42,6 +42,19 @@ export class UmbInputTinyMceElement extends FormControlMixin(UmbLitElement) {
 		return this._editorElement?.querySelector('iframe') ?? undefined;
 	}
 
+	set value(newValue: FormDataEntryValue | FormData) {
+		this._value = newValue;
```
Comment (nielslyngsoe) [inline src/packages/core/components/input-tiny-mce/input-tiny-mce.element.ts:46]: Sadly we dont have any way to enforce calling the super method. But the FormControlMixin has a getter and setter for the value that we need to call.

So this should be changed into: `super.value = newValue`

Comment (nielslyngsoe) [reply src/packages/core/components/input-tiny-mce/input-tiny-mce.element.ts:46]: Otherwise really good! 👍

Comment (enkelmedia) [reply src/packages/core/components/input-tiny-mce/input-tiny-mce.element.ts:47]: @nielslyngsoe So you are saying that 

```
this._value = newValue;
```

should be

```
super.value = newValue;
```

I'll try that and update the PR =D

**Comment by enkelmedia** [comment]
@nielslyngsoe Updated to use super and tested, it seems to work =D

File: src/packages/core/components/input-tiny-mce/input-tiny-mce.element.ts (hunk 1 above)
Comment (enkelmedia) [reply src/packages/core/components/input-tiny-mce/input-tiny-mce.element.ts]: Updated with new commit

**Review by nielslyngsoe — APPROVED** [review]
Looks good, thanks!

**Pull request merged**